exporters/stats/stackdriver/src/main/java/io/opencensus/exporter/stats/stackdriver/StackdriverExporterWorkerThread.java

```diff
@@ -35,16 +37,10 @@
  */
 final class StackdriverExporterWorkerThread extends Thread {
 
-  private static final Object monitor = new Object();
```

Comment:
It would help to add a comment or a NotThreadSafe annotation if this class' state should now only be accessed from the StackdriverExporterWorkerThread thread.

Reply:
That's the case, done.